tests/Behat/Resources/services/pages/product.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <defaults public="true" />
+
+        <service id="sylius_price_history.behat.page.admin.product.show_page" class="Tests\Sylius\PriceHistoryPlugin\Behat\Page\Admin\Product\ShowPage" parent="sylius.behat.symfony_page" public="false" />
```

Comment:
```suggestion
        <service id="sylius_price_history.behat.page.admin.product.show_page" class="Tests\Sylius\PriceHistoryPlugin\Behat\Page\Admin\Product\ShowPage" parent="sylius.behat.symfony_page" public="false" />
```

Reply:
When `defaults public` removed you can always remove `public="false"` as all services are private by default.